python3-zstandard runs out of memeory on i686 with comp_level=20
diff --git a/scripts/pack.py b/scripts/pack.py
index 913e5a66..a7d1d9c0 100755
--- a/scripts/pack.py
+++ b/scripts/pack.py
@@ -104,12 +104,12 @@ def log_file(path, arcname, comp_type=None):
 def pack(args):
     nocompress_file = args.directory / '.nocompress'
 
-    if 1:
+    if sys.maxsize > 2**32:
         comp_type = ZIP_ZSTANDARD
         comp_level = 20
     else:
-        comp_type = ZIP_DEFLATED
-        comp_level = 9
+        comp_type = ZIP_ZSTANDARD
+        comp_level = 19
 
     try:
         nocompress = list(map(re.compile, filter(None, nocompress_file.read_text().strip().split('\n'))))
